docusaurus.config.js

```diff
@@ -79,6 +79,11 @@ const config = {
           }
         ],
       },
+      algolia: {
+        appId: "QIQQLP8ODT",
```

Comment:
Can we store these in an env

Reply:
These all are public keys. So its fine to keep it there